src/Bicep.Core/Parsing/Parser.cs

```diff
@@ -310,8 +312,27 @@ private ImportedSymbolsListSyntax ImportedSymbolsList()
             return new(openBrace, itemsOrTokens, closeBrace);
         }
 
-        private ImportedSymbolsListItemSyntax ImportedSymbolsListItem()
-            => new(Identifier(b => b.ExpectedExportedSymbolName()), ImportedSymbolsListItemAsClause());
+        private SyntaxBase ImportedSymbolsListItem()
+        {
+            SyntaxBase originalSymbolName = reader.Peek().Type switch
+            {
+                TokenType.Identifier => Identifier(b => b.ExpectedExportedSymbolName()),
+                TokenType.StringComplete or
+                TokenType.StringLeftPiece => InterpolableString(),
```

Comment:
Why support interpolation?

Reply:
It was slightly more work to block interpolation than to support it, and I didn't have a compelling reason to choose either alternative. It's probably safer to block interpolation for now until we have a use case for it.